sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -69,18 +77,17 @@ export interface StudentRestrictionDTO {
 /**
  * DTO object for student details.
  */
-export interface StudentDetailDTO {
+export class StudentDetailDTO {
```

Comment:
Just wanted to remind that, if we are changing from interface to class then make sure it is not used in any of the API as payload. If otherwise the payload will not be parsed without using @Allow decorator.

Reply:
it's an OUT DTO. i kept it for the API refactoring story, anyway I will rename it `StudentDetailAPIOutDTO`.